sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -36,6 +37,13 @@ export interface EducationProgramDto {
   programDeclaration: boolean;
 }
 
+export interface EducationProgramData extends EducationProgramDto {
```

Comment:
Please can you keep the prefix/suffix of naming convention here in sync? **EducationProgramData** is a **Dto** which extends a another base **EducationProgramDto**  **Dto** interface that is specific to education program with respect to common attributes. Example, something like  **export interface EducationProgramDataDto extends BaseEducationProgramDto**  will suffice.